CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [2.4.12] - (Unreleased)
 
+### Added
+
+- Added options for readbytes and readtext
```

Comment:
"Added options parameter to readbytes and readtext functions" ?

For symmetry, maybe it's worth modifying writebytes and writetext too?